integration/createdeletebeforeactive_test.go

```diff
@@ -0,0 +1,87 @@
+// +build integration
+
+package integration_test
+
+import (
+	"fmt"
+
+	awseks "github.com/aws/aws-sdk-go/service/eks"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/weaveworks/eksctl/pkg/ctl/cmdutils"
+	"github.com/weaveworks/eksctl/pkg/testutils/aws"
+	. "github.com/weaveworks/eksctl/pkg/testutils/matchers"
+)
+
+const (
+	POLLINTERVAL = 15   //seconds
```

Comment:
convention for Go constants is camel case

Reply:
switched to camelCase